instrumentation/opentelemetry-instrumentation-boto/src/opentelemetry/instrumentation/boto/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+_instruments = ("boto~=2.0",)
```

Comment:
Could you include in `CONTRIBUTING.MD` for the instrumentation checklist that `_instruments` needs to be included?

Reply:
Yes. I'll do it in https://github.com/open-telemetry/opentelemetry-python-contrib/pull/514 as it includes some more changes wrt to contributing instrumentations.